packages/ui/src/main.ts

```diff
@@ -39,11 +40,14 @@ export interface ConnectOptions extends GetWalletOptions {
   storeVersion?: StoreVersion
 }
 
-const enableWithVersion = async (wallet: StarknetWindowObject | null) => {
+const enableWithVersion = async (
```

Comment:
Should get rid of this method and just use connect -> sn.enable()

Reply:
could u pls show with code? 🙏